data/config-schema/src/main/resources/net/corda/schema/configuration/p2p.gateway/1.0/corda.p2p.gateway.json

```diff
@@ -18,6 +18,12 @@
       "minimum": 0,
       "maximum": 65535
     },
+    "urlPath": {
```

Comment:
We could consider combining this with the `hostAddress` and `hostPort` as a URI.

Reply:
[Discussed in the standup] Agreed to keep them as separate fields for now to prevent confusion between binding address/port and connection URL. And to make it a bit more easy for people to only adjust one part without having to specify a full URL. We can revisit later on depending on customer feedback.